autocodesign/devportalclient/appstoreconnectclient/devices.go

```diff
@@ -64,7 +64,7 @@ func (d *DeviceClient) RegisterDevice(testDevice devportalservice.TestDevice) (*
 
 	registeredDevice, err := d.client.Provisioning.RegisterNewDevice(req)
 	if err != nil {
-		rerr := appstoreconnect.ErrorResponse{}
+		rerr := &appstoreconnect.ErrorResponse{}
```

Comment:
In the `profiles.go` file you were using the 
```
var respErr *appstoreconnect.ErrorResponse
```
format at both places and I wonder if we could use the same here as well just for consistency.